src/core/bitop.d

```diff
@@ -72,20 +72,17 @@ unittest
  */
 int bsf(size_t v) pure;
 
+version(CoreDdoc)
 /// ditto
+int bsf(ulong v) pure;
+
+version(D_LP64) { } else
```

Comment:
> And this is what `D_LP64` for. If someone don't know about `D_LP64` it's not our problem.

Looking at this again, I see that you actually missed my point about the "precise bit-ness of the types".

Leaving aside the issue of `static if` verses `version`, the `else static assert(0);` is there because the code which you have marked as "anything but 64-bit" is really intended *specifically* for 32-bit.

So, instead of `version (D_LP64) { } else`, it should really just be `version (D_LP32)` (or the equivalent `static if`).